tests/integration/views/test_request.py

```diff
@@ -443,6 +443,28 @@ def test_request_submit_not_author(airlock_client):
     assert persisted_request.status == RequestStatus.PENDING
 
 
+def test_request_workspace(airlock_client):
+    airlock_client.login(workspaces=["test1"])
+    other_user = factories.create_user("other", ["test1"], False)
```

Comment:
Why not just `user`?

Reply:
it's to make the point that the release requests we're to be about to create in this test are from a different user than the one that we're logged in to `airlock_client` with. That could be clearer, though, I could make it that explicit (by creating a second user in this test & explicitly logging in to `airlock_client` with that user)?

Reply:
also, this same pattern is used in the existing `test_request_submit_not_author(airlock_client)` - should I change that too?